picard/coverart/image.py

```diff
@@ -266,6 +266,12 @@ def save(self, dirname, metadata, counters):
         else:
             filename = config.setting["cover_image_filename"]
             log.debug("Using default cover image filename %r", filename)
+        try:
+            from picard.coverart import run_coverart_file_save_action
+            run_coverart_file_save_action(self)
+        except Exception as e:
+            log.debug(e)
```

Comment:
I think it would be better to use `log.error()` and prefix the message with the hook name. Messages from `log.debug()` aren't shown by default, but here i think we need to let the user know something went wrong with (possibly) one of his plugins.